src/harness/testcases/WINNF_FT_S_MCP_testcase.py

```diff
@@ -476,29 +618,25 @@ def executeSingleMCPIteration(self, test_type, iteration_content, sas_test_harne
     # Step 16 : Trigger CPAS and wait for its completion
     self.TriggerDailyActivitiesImmediatelyAndWaitUntilComplete()
 
-    # Step 17 : Call IAP reference model
```

Comment:
Please add a note saying that this will happen as part of step 22, which is OK.

Reply:
Added the comment